docs/src/pages/components/icons/icons.md

```diff
@@ -152,27 +149,35 @@ import { ReactComponent as StarIcon } from './star.svg';
 <SvgIcon component={StarIcon} viewBox="0 0 600 476.6" />
 ```
 
+### createSvgIcon
+
+The `createSvgIcon` utility component is used to create the [Material icons](#material-icons). It can be used to wrap an SVG path with an SvgIcon component.
```

Comment:
Should we avoid to abbreviate "Material Design" with "Material". Could it be confusing once we provide a second design spec and try to use "Material" with a meaning closer to https://dictionary.cambridge.org/us/dictionary/english/material 😁

> a physical substance that things can be made from

?

```suggestion
The `createSvgIcon` utility component is used to create the [Material Design icons](#material-icons). It can be used to wrap an SVG path with an SvgIcon component.
```

Reply:
If we do (and I honestly don't think we'll ever be able to get away from Material referring to Material Design), then let's do it here then. In the meantime it's consistent with other usage.

Reply:
Well regarding consistency, I have searched for "Material" in the source. "Material Design" seems more popular 😁.

Reply:
"Material icons"
https://material.io/resources/icons/?style=baseline